internal/api/v2/unmarshal.go

```diff
@@ -49,6 +50,10 @@ func chainFromStateObj(obj *state.Object) (state.Chain, error) {
 		chain = new(protocol.KeyBook)
 	case types.ChainTypeTransaction:
 		chain = new(state.Transaction)
+	case types.ChainTypeDataAccount:
+		chain = new(protocol.DataAccount)
+	case types.ChainTypeLiteDataAccount:
+		chain = new(protocol.LiteDataAccount)
```

Comment:
I only added this here for consistency.  I added a new jira task to consolidate the switches

Reply:
:+1: